fastlane/lib/fastlane/actions/git_commit.rb

```diff
@@ -37,7 +37,7 @@ def self.description
       def self.available_options
         [
           FastlaneCore::ConfigItem.new(key: :path,
-                                       description: "The file you want to commit",
+                                       description: "The file(s) you want to commit, use . to match all paths",
```

Comment:
Hi, thank you for taking the time. I'm happy to join you to improve this document. This is a good point that parameter name is singular "path" which doesn't sound like you can commit multiple files. So I like adding "(s)"🚀 

However, I believe that generally "." is a bad practice that you may involve unwanted changes and occurs incidents. (Though I use it. I know this is very handy🙈) The thing is that if you know how this could be bad, you would be fine but given fastlane are used a lot of users including beginners, I think it's not good that spreading a bad practice in its document. For example, this case would be safe because the top `ensure_git_status_clean` prevents you from committing files made between the first and last line. You must be aware of what you are going to commit.

```ruby
ensure_git_status_clean
sh('touch tmp/.gitignore')
git_commit(path: ".")
```

I couldn't come up with a good example but if you were to commit files that are generated by their party tool, it can be dangerous in future as you may not know what file updates will be coming after some version bumps of if.

```ruby
sh('some_third_pary_cli_tool_to_update_some_files_in_project')
git_commit(path: ".")
```

-------------------------

I come to think that instead, we should mention you can use `Array` to specify multiple file paths or fileglobs, which are missing here. Additionally, it's worth mentioning here about `git commit <file path>`'s behaviour in untracked and staged files. 

```suggestion
                                       description: "The file(s) or directory(ies) you want to commit. You can pass an array of multiple file-paths or fileglobls \"*.txt\" to commit all matching files. The files already staged but not specified in this or untracked files won't be committed.",
```

This is what I can think of to suggest better practice to beginners instead.😅 What do you think?

Reply:
Hi, thanks for your reply and feedback, it's much appreciated. I understand your point on not wanting to commit added files, if you haven't specified them. My main motivator for this PR was actually the following error message, which occurs if you specify an empty string for the path parameter:
```
C:/tools/ruby27/lib/ruby/gems/2.7.0/gems/fastlane-2.172.0/fastlane_core/lib/fastlane_core/ui/interface.rb:153:in `shell_error!': \e[31m[!] Exit status of command 'git commit -m "Version bump and release" ""' was 128 instead of 0. (FastlaneCore::Interface::FastlaneShellError)
fatal: empty string is not a valid pathspec. please use . instead if you meant to match all paths
\e[0m
```
This sort of ends up recommending that you use "." anyway (although I understand it's just passing up an error from the shell). 

I'll commit your suggestion. I may open a second PR to catch the empty path parameter earlier and raise a custom error (perhaps suggesting that you can use "." but recommending you not to, using your reason above?). What do you think about that idea?

Thanks again! :)

Reply:
I understood how you came across "."😓 and the plan sounds good to me! 

> perhaps suggesting that you can use "." but recommending you not to, using your reason above?

Maybe we should still suggest passing file paths *first* and then the above can follow it. An empty string for `path` can be a program error if it's given from some logic.